src/main/resources/application.properties

```diff
@@ -3,6 +3,8 @@ DOMAIN=localdomain
 OAUTH_CLIENT_ID=null
 OAUTH_SECRET=null
 
+server.port=8090
```

Comment:
Not sure how much stuff in here we'll want to change.  Changing this port might screw up PCF.  Also, for these default usernames and passwords, did those change in our vagrant VMs?  Usually it's nice if those line up.

Reply:
Great catch. That thing was all messed up from my testing environment.
I restored to the previous version then added back the geoserver style/layer name changes.

Reply:
Yep, let's remove server.port.